diff -Naupr a/qt5/style/qtcurve.cpp b/qt5/style/qtcurve.cpp
--- a/qt5/style/qtcurve.cpp	2017-06-01 18:01:53.000000000 +0200
+++ b/qt5/style/qtcurve.cpp	2018-09-13 16:55:55.305575870 +0200
@@ -702,9 +702,6 @@ void Style::disconnectDBus()
         return;
     m_dbusConnected = false;
     auto bus = QDBusConnection::sessionBus();
-    if (getenv("QTCURVE_DEBUG")) {
-        qWarning() << Q_FUNC_INFO << this << "Disconnecting from" << bus.name() << "/" << bus.baseService();
-    }
     bus.disconnect(QString(), "/KGlobalSettings", "org.kde.KGlobalSettings",
                    "notifyChange",
                    this, SLOT(kdeGlobalSettingsChange(int, int)));
